akd_local_auditor/src/storage/dynamodb/test.rs

```diff
@@ -0,0 +1,346 @@
+// Copyright (c) Meta Platforms, Inc. and affiliates.
+//
+// This source code is licensed under both the MIT license found in the
+// LICENSE-MIT file in the root directory of this source tree and the Apache
+// License, Version 2.0 found in the LICENSE-APACHE file in the root directory
+// of this source tree.
+
+//! A note to readers:
+//!
+//! We are marking most of the following tests as `#[ignored]` so they don't run automatically
+//! with `cargo test`. These require a docker container running `minio` to be running which is
+//! run in our CI pipeline when changes to this crate are made, but not otherwise since they
```

Comment:
With ignore they will be ignored regardless of where they are run, right? So we need to enable them later. Maybe create an issue for it?

Reply:
The AWS CI pipeline (`aws.yml`) will run them specifically with the --ignored flag in cargo. If you take a look there, you can see it only triggers on file-changes in a specific code path which then runs these more expensive tests. If no one changes this crate, the tests don't run to save cycles.